helios-client/src/main/java/com/spotify/helios/client/tls/X509CertificateFactory.java

```diff
@@ -83,8 +89,12 @@ public static Certificate get(final AgentProxy agentProxy, final Identity identi
           .get(new AlgorithmIdentifier(OIWObjectIdentifiers.idSHA1));
       final X509ExtensionUtils utils = new X509ExtensionUtils(digestCalculator);
```

Comment:
Unfortunately no since the creation of the `DigestCalculator` can throw an exception, and that doesn't play nice with static constructors.